components/ws-daemon/pkg/content/service.go

```diff
@@ -475,7 +471,7 @@ func (s *WorkspaceService) uploadWorkspaceContent(ctx context.Context, sess *ses
 		}()
 
 		var opts []archive.TarOption
-		if sess.UserNamespaced && !sess.FullWorkspaceBackup {
+		if !sess.FullWorkspaceBackup {
```

Comment:
When we bring this into production, we need to think about the non-user-namespaced workspaces that are currently running and will be backed up by this new version of ws-daemon. We should maintain the different behaviour in ws-daemon until after the next deployment (first week task to clean things up).